x-pack/plugins/observability_solution/dataset_quality/public/state_machines/dataset_quality_controller/src/types.ts

```diff
@@ -101,14 +103,6 @@ export type DatasetQualityControllerTypeState =
       value: 'datasets.loaded.idle';
       context: DefaultDatasetQualityStateContext;
     }
-  | {
```

Comment:
why we don't need these anymore?

Reply:
I couldn't find the states corresponding `'datasets.loaded.flyoutOpen.fetching'` and `'datasets.loaded.flyoutOpen'` in any machine.